src/mutate/matchUps/matchUpStatus/setMatchUpState.ts

```diff
@@ -534,17 +535,19 @@ function checkParticipants({
 
   if (
     matchUpStatus &&
-    [WALKOVER, DEFAULTED].includes(matchUpStatus) &&
+    //we want to allow wo, default and double walkover inn the consolation draw
+    //to have only one particpiant when they are caused by an exit propagation 
+    [WALKOVER, DEFAULTED, DOUBLE_WALKOVER].includes(matchUpStatus) &&
```

Comment:
@CourtHive  this is needed because if we get a double WO from the main draw, that is propagated as a WO without a participant in the consolation match. If we then assign another WO for the opponent in that consolation match, we end up with two WO but only one participant and we then want to set that match as a DOUBLE WALKOVER.